res/graphics/quest/steps_brown.svg

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Please keep the Android drawable filename and the SVG source filename in sync, so they can be matched in the quest list generation task (and by humans).

Reply:
yes, I used wrong icon as a base